validator/node/node.go

```diff
@@ -101,6 +101,7 @@ func NewValidatorClient(ctx *cli.Context) (*ValidatorClient, error) {
 		if err := clearDB(dataDir, pubkeys, forceClearFlag); err != nil {
 			return nil, err
 		}
+		log.WithField("path", dataDir).Info("Checking DB path")
```

Comment:
Can you mirror this to the beacon node log? So `Checking DB` and the field name `databasePath`.

Reply:
Okay, done